common/ethereum/src/eip_4844.rs

```diff
@@ -0,0 +1,58 @@
+use common::types::Result;
+use common_chain_ids::EthChainId;
+use derive_more::Constructor;
+use ethereum_types::U256;
+
+use crate::EthBlock;
+
+#[derive(Clone, Constructor)]
+pub struct Eip4844 {}
+
+impl Eip4844 {
+    #[allow(dead_code)]
```

Comment:
Might as well make this:

```suggestion
    #[cfg(test)]
```

instead.

Reply:
That function should be removed because it is not used and it will be not used until 66193ff is reverted. 

Replacing with `#[cfg(test)]`, clippy warns us with `unused import: ethereum_types::U256` therefore the changes should be 
```suggestion
    #[cfg(test)]
    pub fn get_activation_block_number(&self, eth_chain_id: &EthChainId) -> Result<U256> {
        use ethereum_types::U256;
```

I would prefer to remove the function instead

Reply:
Once we know the activation block number, we may already be past it so it becomes pointless.

Your way of checking the block's contents is sufficient for us to be ready in advance without having to do anything further. 

In fact, it's so sufficient we might as well ditch the activation number thing altogether.

The worse case scenario is an attacked trying to submit a block that's not EIP4844 compliant, but with those fields added, at which point it'd fail validation anyway so I see no danger there.